languages/javascript/exercises/concept/strings/.docs/after.md

```diff
@@ -0,0 +1,5 @@
+The key thing to remember about JavaScript strings is that they can call methods like `toUpperCase`, `trim`, etc.
```

Comment:
The wording `they can call` is little confusing - maybe `we can call methods on them like...`

Reply:
I did.